Dfe.Academies.External.Web/Pages/ApplicationOverview.cshtml.cs

```diff
@@ -116,7 +123,18 @@ private void PopulateUiModel(ConversionApplication? conversionApplication)
 		{
 			
 			// grab current user email
-			string email = User.FindFirst(ClaimTypes.Email)?.Value ?? "";
+			string email = User.FindFirst(ClaimTypes.Email)?.Value ?? "";		
+			var firstContributorEmail = conversionApplication.Contributors.FirstOrDefault().EmailAddress;
```

Comment:
This can be null on one or more paths, what is the fallback?

Reply:
neither can be null otherwise the person wouldnt be logged in